<sys/sysctl.h> does not exist in the NDK, so never include it in libiberty.

--- a/libiberty/physmem.c
+++ b/libiberty/physmem.c
@@ -48,7 +48,7 @@
 # include <sys/param.h>
 #endif
 
-#if HAVE_SYS_SYSCTL_H
+#if false
 # include <sys/sysctl.h>
 #endif
 
